lib/get-pattern-import-data.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+var path = require('path'),
+  fs = require('fs'),
+  utility = require('./utility');
+
+/*
+ * Gets details on an html pattern (and its supporting files) coming from a Pattern Library-structured directory
+ *
+ * @param {Array} paths  path info on pattern
+ * @param {Object} options  pattern-importer options
+*/
+exports.getPatternImportData = function (paths, options) {
+
+  // open the individual pattern's data file
+  var patternYml = fs.readFileSync(path.join(paths.folder, options.dataFileName), {encoding:'utf8'});
+
+  // create an object to store data about this pattern's files
+  var patternFiles = {};
+  patternFiles.data = {};
+  patternFiles.filesToWrite = [];
```

Comment:
@sergesemashko 
instead of filesToWrite/filesToCopy we should probably set up our vinyl streams for these files, correct?